Packages/Core/Editor/Generator/Generator.cs

```diff
@@ -58,7 +58,11 @@ public void Generate(string type, string baseWritePath, bool isEquatable, List<A
             var templateSearchPath = Runtime.IsUnityAtomsRepo ?
                 Directory.GetParent(baseWritePath).FullName : // "Packages"
                 Directory.GetParent(Application.dataPath).FullName;
+#if UNITY_2018_4_OR_NEWER
```

Comment:
Same as above. The UNITY_2019_1_OR_NEWER condition should be before UNITY_2018_4_OR_NEWER, the second if-statement will never be used when running Unity 2019 since the UNITY_2018_4_OR_NEWER flag will also be true in this case.